packages/icons/docs/stories/rainbowIconItem.js

```diff
@@ -0,0 +1,135 @@
+import React, { useRef, useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import InternalOverlay from 'react-rainbow-components/components/InternalOverlay';
+import RenderIf from 'react-rainbow-components/components/RenderIf';
+import useWindowResize from 'react-rainbow-components/libs/hooks/useWindowResize';
+import useUniqueIdentifier from 'react-rainbow-components/libs/hooks/useUniqueIdentifier';
+import { StyledTooltip, StyledTitle, StyledText, StyledButton, StyledSample } from './styled';
+
+const RainbowIconItem = (props) => {
+    const { rainbowIcon, id } = props;
+
+    const triggerRef = useRef();
+    const helpTextId = useUniqueIdentifier('help-text');
+    const [isFocused, setIsFocused] = useState(false);
+    const isHoverTooltip = useRef(false);
+    const isClickTooltip = useRef(false);
+    const [isOpen, setIsOpen] = useState(false);
+    const closeOverlay = useCallback(() => setIsOpen(false), []);
+    const openOverlay = useCallback(() => setIsOpen(true), []);
+    const ESCAPE_KEY = 27;
```

Comment:
this const can be defined outside the component thus avoiding the const to be recreated in each render